.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+node-options="--openssl-legacy-provider"
```

Comment:
Fixes this issue 👇 

```
 npm install --legacy-peer-deps && npm run build

up to date, audited 2380 packages in 24s

197 packages are looking for funding
  run `npm fund` for details

92 vulnerabilities (2 low, 66 moderate, 21 high, 3 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.

> build
> wp-scripts build --config webpack.config.js --mode production

Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
node:internal/crypto/hash:69
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:69:19)
    at Object.createHash (node:crypto:133:10)
    at module.exports (/Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/webpack/lib/util/createHash.js:135:53)
    at NormalModule._initBuildHash (/Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/webpack/lib/NormalModule.js:417:16)
    at handleParseError (/Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/webpack/lib/NormalModule.js:471:10)
    at /Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/webpack/lib/NormalModule.js:503:5
    at /Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/webpack/lib/NormalModule.js:358:12
    at /Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/loader-runner/lib/LoaderRunner.js:373:3
    at iterateNormalLoaders (/Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
    at Array.<anonymous> (/Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
    at Storage.finished (/Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
    at /Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9
    at /Users/dntvngp/Documents/Greenpeace/International/Projects/planet4-develop/planet4/themes/planet4-master-theme/node_modules/graceful-fs/graceful-fs.js:123:16
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}

Node.js v18.20.4
```

Reply:
And can be potentially removed after https://github.com/greenpeace/planet4-master-theme/pull/2188

Reply:
It's missing an empty EOL

Reply:
Do you think it's neccesary?

I'm getting this warning message
```
<stdin>:8: new blank line at EOF.
+
warning: 1 line adds whitespace errors.
```
I think that the editorconfig solves this whitin [this](https://github.com/greenpeace/planet4-master-theme/blob/2cff54462fe197e73492e80a99a9f6ca24d0c616/.editorconfig#L11-L12). Also there some default files that does not have EOL, like the [package.json](https://github.com/greenpeace/planet4-master-theme/blob/main/package.json#L57) also some blocks, like the [covers block](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/blocks/Covers/CoversBlock.js#L73)

Reply:
Maybe you're right!

Reply:
But I see that you fixed it in the [plugin PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1260/files)

Reply:
Yes, but only for testing purposes. I prefer to keep the same as the original version. Do you agree?

Reply:
As you prefer, I don't have strong feelings about it 🙂 I mostly reported it because Github highlights it

Reply:
In addition, that's why I preferred to define the rest of parameters into an object, and than just pass whatever you need.

`const searchAndInsertBlock = async ({page, admin, editor}, {blockName, namespace}) => {`

Reply:
I think this comment is not in the right thread @GP-Dan-Tovbein ?

Reply:
You're correct! Sorry, too many PRs opened 🤦